tests/Crypto/tst_cryptorequests/tst_cryptorequests.cpp

```diff
@@ -188,6 +221,10 @@ private slots:
         QTest::newRow("GCM 192-bit") << CryptoManager::AlgorithmAes << CryptoManager::BlockModeGcm << CryptoManager::EncryptionPaddingNone << 192;
         QTest::newRow("GCM 256-bit") << CryptoManager::AlgorithmAes << CryptoManager::BlockModeGcm << CryptoManager::EncryptionPaddingNone << 256;
 
+        QTest::newRow("CCM 128-bit") << CryptoManager::AlgorithmAes << CryptoManager::BlockModeCcm << CryptoManager::EncryptionPaddingNone << 128;
```

Comment:
AES CCM 128-bit, probably same for the GCM cases above (and are the CTR cases above that duplicates, should be removed?)